modules/livestore/live_store.go

```diff
@@ -259,8 +265,15 @@ func (s *LiveStore) starting(ctx context.Context) error {
 		return fmt.Errorf("failed to start partition reader: %w", err)
 	}
 
-	s.wg.Add(1)
-	go s.completeLoop()
+	for i := range s.cfg.CompleteBlockConcurrency {
+		idx := i
```

Comment:
protecting against the ye olde reassignment issue? Though I think its fixed now.